buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -240,14 +240,20 @@ abstract class AndroidTarget extends JavaLibTarget {
     }
 
     String processManifestXml(GPathResult manifestXml) {
-        def sdkNode = {
-            'uses-sdk'('android:minSdkVersion': String.valueOf(minSdk),
-                    'android:targetSdkVersion': String.valueOf(targetSdk)) {}
-        }
         if (manifestXml.'uses-sdk'.size() == 0) {
-            manifestXml.appendNode(sdkNode)
+            manifestXml.appendNode {
```

Comment:
Thats nice. We can make the logic simpler here by constructing the sdkNode correctly all the time i.e read any and all existing attrs if the node exists, and add our modifications on top when creating the new node to always replace existing.

How does that sound? Then it is no loner specific to `tools:overrideLibrary` and will work for all attributes